src/cli/run_command.zig

```diff
@@ -1420,41 +1313,80 @@ pub const RunCommand = struct {
             return true;
         }
 
-        // Run package.json script
+        // check for stdin
+
+        if (target_name.len == 1 and target_name[0] == '-') {
+            log("Executing from stdin", .{});
+
+            // read from stdin
+            var stack_fallback = std.heap.stackFallback(2048, bun.default_allocator);
+            var list = std.ArrayList(u8).init(stack_fallback.get());
+            errdefer list.deinit();
+
+            std.io.getStdIn().reader().readAllArrayList(&list, 1024 * 1024 * 1024) catch return false;
```

Comment:
let's avoid using zig std for this, they use too much unreachable for this to not cause issues for us.

instead, use bun.sys.File